dockers/docker-lldp-sv2/config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/bin/cp -rf /etc/swss/lldp/lldpd.j2 /etc/default/lldpd
```

Comment:
can we add -e, to exit when there is failure. otherwise this script will always just return success.

---
In reply to: [96062158](https://github.com/Azure/sonic-buildimage/pull/190#discussion_r96062158) [](ancestors = 96062158,96061953)